### DISCUSSION TITLE
non editable customer facing data table embeds

### DISCUSSION THREAD

**Discussion by jay377** [nocodb/nocodb#1020] [Q&A] [answered]
Hello. Can I use this to create simple non editable customer facing data tables that pulls in data from my local DB or via API from another system? So no editing functions or other views, I only want to see a datable that I can embed in another webpage. Thanks!

**Comment by wingkwong** [comment, marked as answer]
You may try Share View which is non-editable.

![image](https://user-images.githubusercontent.com/35857179/151652227-71eab955-281f-42a4-a948-a60b71ab5d9f.png)

**Reply by jay377** [reply]
That looks great thanks! In terms of showing data for specific users, is it possible to send filter or parameter data to the embed to show only specific data depending on the page the embed is on or what user is viewing the shared view? Is there some way to show different data to different users? Thanks

**Reply by wingkwong** [reply]
In this case, you can create different views. 

For example, here I created a new one called ``FilmWithTitleOnly`` and I only select ``Title`` field.

![image](https://user-images.githubusercontent.com/35857179/151653758-92e2cfb5-5743-45fe-8b01-25c092799f93.png)

For each view, you could have a dedicated share view link.

![image](https://user-images.githubusercontent.com/35857179/151653774-03ecad01-fb23-4732-9535-a97810b60135.png)
